c/include/libsbp/new/sbp_msg.h

```diff
@@ -52,42 +52,40 @@ extern "C" {
 struct sbp_state;
 
 typedef union {
-  sbp_msg_tracking_state_detailed_dep_a_t tracking_state_detailed_dep_a;
```

Comment:
Weird, I don't think there's actually any diff between these two branches on this file. Maybe a Github bug?